src/infra/messaging-outbox/sqlite/src/repos/sqlite_outbox_message_repository.rs

```diff
@@ -93,6 +94,7 @@ impl OutboxMessageRepository for SqliteOutboxMessageRepository {
                     occurred_on
                 FROM outbox_messages
                 WHERE {producer_filters}
+                and version = {OUTBOX_MESSAGE_VERSION}
```

Comment:
Minor: `indoc::formatdoc!()` can be used in this (and similar cases) -- it is a combination of the indoc::indoc!() and format!() macros

Reply:
Looks good. Will use it